api/envoy/config/cluster/redis/BUILD

```diff
@@ -1,7 +1,10 @@
-load("@envoy_api//bazel:api_build_system.bzl", "api_proto_library_internal")
+load("@envoy_api//bazel:api_build_system.bzl", "api_proto_library_internal", "api_proto_package")
 
 licenses(["notice"])  # Apache 2
 
+api_proto_package(
```

Comment:
Can you do a pass to make these a bit more elegant, e.g. this should be `api_proto_package()`, below should be:
```
api_proto_package(
    deps = ["//envoy/apiv2"]
)
```

Reply:
Reformatted to make it prettier.